src/deploy/AWSDeployer.js

```diff
@@ -305,6 +317,25 @@ class AWSDeployer extends BaseDeployer {
     }
   }
 
+  async updatePackage() {
+    this.log.info('--: updating app (package) parameters …');
```

Comment:
can we please not use the `…` but rather `...` ? it really looks odd in the console, like a stray underscore.